docs/content/Configuration/Advanced/Multiple-Data-Sources.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Multiple Data Sources
+permalink: /config/multiple-data-sources
+category: Configuration
+subCategory: Advanced
+menuOrder: 2
+---
+
+Cube supports using multiple data sources, allowing the use of different
+databases for different cubes.
+
+## Schema
+
+To configure multiple data sources, first explicitly declare a data source in
+some cubes by adding a `dataSource` property:
+
+```javascript
+cube('Orders', {
+  ...,
+
+  dataSource: 'default',
+});
+
+cube('OrdersFromOtherDataSource', {
+  ...,
+
+  dataSource: 'datasource1',
+});
+```
+
+Next, Cube must be configured so that it knows which database to use for each
+data source. This can be done either with the
+[`driverFactory`][ref-config-ref-driverfactory] property in the `cube.js`
+configuration file or through environment variables.
+
+## Configuring data sources with cube.js
```

Comment:
I think we want to focus only on env based management

Reply:
I thought it would be good to contrast the approaches, especially since there's content in the existing Multitenancy page which really belongs here too.